### PR TITLE
docs(pt): update and improve the readability of the content

### PR DESCRIPTION
**Pull request by nazarepiedady**
In this pull request, I am sending impactful updates to the
readability of the Portuguese content to ease even more  
the learning process of the framework without too much mental 
effort to understand the content.

@smarroufin and @ChristopheCVB

### CODE REVIEW

**Review by smarroufin — CHANGES_REQUESTED** [review]
Thanks for the massive update!
I commented a few things to adjust.
Also, don't update the vue router links, as they are correct with the v3 subdomain. Nuxt 2 is using Vue Router 3, so we should not send to v4 documentation.

**Comment by nazarepiedady** [comment]
@smarroufin the points marked were resolved.

**Comment by smarroufin** [comment]
@nazarepiedady You still need to rollback the vue router links, keep the `v3.` in the url.

**Comment by nazarepiedady** [comment]
@smarroufin recently some updates were added to the repository that changed the
URLs to Nuxt 3 to this form that I used `https://nuxt.com`, so is it still necessary to do the change back?

**Comment by smarroufin** [comment]
This repository has nothing to do with Nuxt v3, and so with Vue Router v4. All the docs here is Nuxt v2 and Vue Router v3, and it shouldn't change.

**Comment by nazarepiedady** [comment]
> This repository has nothing to do with Nuxt v3, and so with Vue Router v4. All the docs here is Nuxt v2 and Vue Router v3, and it shouldn't change.

@smarroufin I understand, so I will do the possible to adjust all to be well done.

**Comment by nazarepiedady** [comment]
@smarroufin and @ChristopheCVB I am going to update and improve the content,
so any problem was found as before I am open to fixing it.

**Comment by smarroufin** [comment]
All good for me, except the updates on VueRouter urls that need to be reverted:
CleanShot 2023-01-24 at 10 57 54

And waiting for @ChristopheCVB for protuguese approval.

File: content/pt/docs/1.get-started/1.installation.md
Comment (ChristopheCVB) [inline content/pt/docs/1.get-started/1.installation.md:20]: delete `de`?

Comment (ChristopheCVB) [reply content/pt/docs/1.get-started/1.installation.md:20]: or put back `de forma`

File: i18n/pt-PT.js
Comment (ChristopheCVB) [inline i18n/pt-PT.js:67]: delete `por`?

File: content/pt/docs/4.directory-structure/12.store.md
Comment (ChristopheCVB) [inline content/pt/docs/4.directory-structure/12.store.md:206]: `memória`

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]
Minor fixes expected

File: content/pt/docs/4.directory-structure/12.store.md
Comment (nazarepiedady) [reply content/pt/docs/4.directory-structure/12.store.md:206]: @ChristopheCVB thanks for noting that, I have the habit write quickly,
and some words go out with some characters missing.

File: i18n/pt-PT.js
Comment (nazarepiedady) [reply i18n/pt-PT.js:67]: Is it really necessary?
I want to be more clear as possible,
in this case, I think it would be better if I use `pelo` instead of `por` or `através do`.

What do you think?

**Comment by nazarepiedady** [comment]
@ChristopheCVB the problems found until now are solved.
I am waiting for your feedback and possibly new issues to solve.

**Comment by nazarepiedady** [comment]
@smarroufin and @ChristopheCVB there are many pull requests
being merged before this one, those that touch on the files I am
working on and when I have to update the local repository to send
you an updated changes those updates insert too many conflicts to
enhancements.

@smarroufin things are becoming too hard for me to have to wait
so long to have the pull requests merged to the main branch. :sweat_smile:

**Comment by smarroufin** [comment]
@nazarepiedady Totally agree, but due to the size of the PR and our synchronization, we are doing our best. @ChristopheCVB is contributing on his free time and sometimes can't review before a while.

Anyways, the PR can't be merged with the Vue Router links you changed. You still need to put back the `v3.` prefix on them (there are 13 replacements to make).
For the portuguese translation, still waiting for an approval.

**Comment by nazarepiedady** [comment]
@smarroufin I thought I already did this, do the issue still remain? :sweat_smile:

**Comment by smarroufin** [comment]
You did fix two wrong urls, but there still are 13 to fix.

**Comment by nazarepiedady** [comment]
Okay, I will fix them too.

File: content/pt/docs/2.concepts/5.nuxt-lifecycle.md
Comment (ChristopheCVB) [inline content/pt/docs/2.concepts/5.nuxt-lifecycle.md:59]: I've you use `o Nuxt` e `a Nuxt`, please stick to the non gender form

File: content/pt/docs/1.get-started/1.installation.md
Comment (ChristopheCVB) [inline content/pt/docs/1.get-started/1.installation.md:9]: To non gender form everywhere for `Nuxt`

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]
Please us e a non gender form for Nuxt

File: content/pt/docs/2.concepts/5.nuxt-lifecycle.md
Comment (nazarepiedady) [reply content/pt/docs/2.concepts/5.nuxt-lifecycle.md:59]: @ChristopheCVB it is impossible, there are no ways to write
about some tool, person, animal, or thing in a more objective
way in Portuguese without using any definite article. :sweat_smile: 

I am using `a` because I am presenting the Nuxt as "a tool",
translated as "uma ferramenta" and as "a framework",
translated as "uma abstração".

Portuguese is not English, and all these chances are to put the
things to sound a little more Portuguese. :point_up:

Comment (ChristopheCVB) [reply content/pt/docs/2.concepts/5.nuxt-lifecycle.md:59]: If you think not using a gender is not correct, then fallback to the masculine.
Here is a nice post about genders in portuguese https://rockcontent.com/br/talent-blog/linguagem-neutra-de-genero/

**Comment by nazarepiedady** [comment]
@smarroufin and @ChristopheCVB, I am closing this pull request in order to start a sequence of small and concise pull request that can facilitate the review.

**Pull request closed without merging**
